topics/statistics/tutorials/regression_machinelearning/data-library.yaml

```diff
@@ -11,18 +11,18 @@ items:
   items:
   - name: Regression in Machine Learning
     items:
-    - name: 'DOI: 10.5281/zenodo.2545213#.XEWTJ9-YVa0'
+    - name: 'DOI: 10.5281/zenodo.2545213'
       description: latest
       items:
-      - url: https://zenodo.org/api/files/f1431fd7-a9dc-426d-ac78-3eaaa61a8d29/test_rows.csv
+      - url: https://zenodo.org/record/2545213/files/test_rows.csv?download=1
         src: url
         ext: csv
-        info: https://zenodo.org/record/2545213#.XEWTJ9-YVa0
-      - url: https://zenodo.org/api/files/f1431fd7-a9dc-426d-ac78-3eaaa61a8d29/test_rows_labels.csv
+        info: https://zenodo.org/record/2545213
+      - url: https://zenodo.org/record/2545213/files/test_rows_labels.csv?download=1
```

Comment:
the download=1 can also be removed I think

Reply:
if download=1 is removed, the file is shown just like that ... that's the error .. the link should automatically download the file

Reply:
We generally do not include it in other datasets. This supports both the usecases of :

- the user to viewing the dataset online without downloading a file and needing to open
- it to be loaded into galaxy.

There are other examples of datasets in libraries that lack the download=1, so, if it is failing, then it is likely due to some other reason.